app-services/secrets/res/configuration.toml

```diff
@@ -1,30 +1,36 @@
 [Writable]
-  LogLevel = 'INFO' 
-  
+  LogLevel = 'INFO'
+
   [Writable.StoreAndForward]
     Enabled = false
     RetryInterval = '5m'
     MaxRetryCount = 10
 
-   [Writable.InsecureSecrets]    
+   [Writable.InsecureSecrets]
       [Writable.InsecureSecrets.NoPath]
         Path = ''
         [Writable.InsecureSecrets.NoPath.Secrets]
           username = 'nopath-user'
           password = 'nopath-pw'
-      
+
       [Writable.InsecureSecrets.AWS]
         Path = '/aws'
         [Writable.InsecureSecrets.AWS.Secrets]
           username = 'aws-user'
           password = 'aws-pw'
-      
+
       [Writable.InsecureSecrets.MongoDB]
         Path = '/mongodb'
         [Writable.InsecureSecrets.MongoDB.Secrets]
           username = ''
           password = ''
 
+      [Writable.InsecureSecrets.Redis]
+        Path = '/redis'
+        [Writable.InsecureSecrets.Redis.Secrets]
+          username = ''
+          password = ''
+
```

Comment:
Need reviewers to confirm if MongoDB and Redis secrets can both be defined in this way.

Reply:
Yes, they can. The path needs to be `redisdb`  w/o  `/`.  `/` also needs to be removed from `/mongodb`